libreflexxestype2/CMakeLists.txt

```diff
@@ -18,43 +22,37 @@ if(CMAKE_DPKG)
 endif()
 
 if(SUPPORTED MATCHES "true")
- 
+  # Set location of build output
+  set(libreflexxes_LOCATION ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so)
+
   # Build Reflexxes library with a Make file
   add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so
-    COMMAND make -C "${CMAKE_CURRENT_SOURCE_DIR}/Linux" clean64 all64 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}  
+    OUTPUT ${libreflexxes_LOCATION}
+    COMMAND make -C "${CMAKE_CURRENT_SOURCE_DIR}/Linux" clean64 all64 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   )
+  add_library(reflexxes ${libreflexxes_LOCATION})
+  set_target_properties(reflexxes PROPERTIES LINKER_LANGUAGE CXX)
 
-  # Export Reflexxes library for ROS
-  add_library(reflexxestype2 ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so)
-  target_include_directories(reflexxestype2 PUBLIC
-    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-    "$<INSTALL_INTERFACE:include>"
+  # Copy into devel/lib
```

Comment:
Nitpick: it's not devel/lib in ROS2. I'll push a commit to clean up this comment